src/ontology/mondo-edit.obo

```diff
@@ -360284,7 +360284,7 @@ xref: GARD:20072 {source="MONDO:GARD"}
 xref: ICD10CM:G40.2 {source="Orphanet:166299", source="Orphanet:166299/attributed", source="Orphanet:166299/ntbt"}
 xref: icd11.foundation:147618459 {source="MONDO:equivalentTo"}
 xref: MEDGEN:1654146 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: Orphanet:166299 {source="MONDO:equivalentTo"}
+xref: Orphanet:166299 {source="MONDO:equivalentObsolete"}
```

Comment:
This is not obsolete in Orphanet: https://www.orpha.net/en/disease/detail/166299